recommender/services/engine_selector.py

```diff
@@ -114,28 +120,32 @@ def engine_loader(
     return engine, eg_name
 
 
-def load_engine(json_dict: dict) -> Tuple[BaseInferenceComponent, str]:
+def load_engine(json_dict: dict) -> Tuple[Any, str]:
```

Comment:
Shouldn't the fallback to AnonymnousInferenceComponent in case of absence of other components be also here?

Reply:
Could you elaborate on that a little bit more? 
The random engine is used mainly when:
- engine_version=random,
- user is anonymous,
- there is no other model saved,
- ...

Reply:
okay